randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;.\n\n"
+            "Additionally, in most game versions, these rocks will not reappear after being destroyed."
+        ),
+    ],
+    "3269646811": [
+        # Reflecting Pool PB Talloric Alloy Tip
+        # Replaces Reflecting Pool lid scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Talloric Alloy&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "1876916293": [
+        # Fungal Hall B PB Tip
+        # Replaces MQB lower right computer screen scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;The item within Fungal Hall B's mushroom ring can be uncovered with "
+            "&main-color=#FF3333;Morph Ball Bombs&main-color=#89D6FF; or "
+            "&main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "4186358462": [
+        # Life Grove Tunnel PB Tip
+        # Replaces Life Grove & LGT Bendezium rock scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;This &main-color=#FF3333;Bendezium&main-color=#89D6FF; surface has a note:\n\n\n "
+            "Life Grove and Life Grove Tunnel each contain a &main-color=#FF3333;Sandstone&main-color=#89D6FF; "
+            "structure.\n\nLike most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "they can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "323204925": [
+        # Watery Hall Access Wall Tip
+        # Replaces Watery Hall Access holobanner scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;In addition to missiles, the weak wall section in this room is able to be destoyed "
+            "by the &main-color=#FF3333;Charge Beam&main-color=#89D6FF;."
+        ),
+    ],
+    "56434148": [
+        # Central Dynamo Unlocked Door Tip
+        # Replaces Central Dynamo lower right computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        "&main-color=#89D6FF;Fighting the Cloaked Drone locks all doors except the door to Quarantine Access A.",
+    ],
+    "3355561537": [
+        # Central Dynamo IS Tip
+        # Replaces Central Dynamo lower left computer screen scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;A corner of this room can be used to acquire &main-color=#FF3333;Infinite Speed"
```

Comment:
It would be worth explaining that it takes advantage of the crates. You may also consider explaining how infinite speed is actually obtained.

Reply:
I think explaining IS will take more space that I'm comfortable using on that trick, but I'd like to dedicate some time in the future to writing up an IS guide to spread across a couple terminals (maybe in this same room if there're scan points available.) The crate mention is good, though.